tests/test_application.py

```diff
@@ -0,0 +1,69 @@
+import requests
+
+import psycopg2
+import pytest
+import testing.postgresql
+from pytest_localserver.http import WSGIServer
+
+from pageview_digest.wsgi import application, db_user, db_passwd, db_dbname
+
+
+@pytest.mark.usefixtures("setup_postgresql")
+class TestWSGIApplication:
+    @pytest.fixture()
+    def setup_postgresql(self):
+        self.postgresql = testing.postgresql.Postgresql(port=5432)
+        conn = psycopg2.connect(**self.postgresql.dsn())
+        conn.autocommit = True
+        cursor = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+
+        # Create configured user if doesn't exist
+        cursor.execute("SELECT COUNT(*) FROM pg_roles WHERE rolname='{}'".format(db_user))
+        if cursor.fetchone()[0] <= 0:
+            cursor.execute("CREATE USER {0} WITH PASSWORD '{1}'".format(
+                db_user, db_passwd
+            ))
+
+        # Create configured database
+        cursor.execute(
+            "SELECT COUNT(*) FROM pg_database WHERE datname='{}'".format(db_dbname)
+        )
+        if cursor.fetchone()[0] <= 0:
+            cursor.execute("CREATE DATABASE {}".format(db_dbname))
+
+        conn.close()
```

Comment:
yes. it is a different database. The initial connection sets up the postgres database and allows me to insert the user/password & database we actually use. 34 - 46 sets up a new connection with the database we actually want the tables from `tables.sql` in